### ISSUE TITLE
Inform people that whitespace at start of code chunks breaks figure references 

### ISSUE THREAD

**Issue by NeuroShepherd** [lmu-osc/introduction-to-Quarto#11]
Works:

````
```{r}
#| label: fig-example
```
````

Won't work:

````
```{r}

#| label: fig-example
```
````

**Comment by NeuroShepherd** [comment]
Remind people that the `fig-` prefix is necessary too. (Also, confirm that this is actually true.)

**Comment by FlorianPargent** [comment]
solved in [22d2d16](https://github.com/lmu-osc/introduction-to-Quarto/commit/22d2d16cc24260f719f3933f6951e986acc79cc2)
